src/php/inc/helpers/generic.php

```diff
@@ -7,7 +7,8 @@
  * Define Globals for within Theme.
  */
 function define_theme_globals() {
-	define( 'GOOGLE_API_KEY', getenv( 'GOOGLE_API_KEY' ) );
+	// phpcs:ignore
+	define( 'GOOGLE_API_KEY', $_SERVER['GOOGLE_API_KEY'] ?? '' );
```

Comment:
I'm having trouble validating this part, `GOOGLE_API_KEY` is defined if I `var_dump` it only within the scope of the function (not elsewhere in `generic.php`), did I miss a step? I did add `GOOGLE_API_KEY` with a value to my .env.

Reply:
I think I was also testing within the scope of that function, and it may not work elsewhere in `generic.php` because of when the 'init' filter runs. If you `var_dump` on a different page, like `index.php`, does it work there, @rise-erpelding?

Reply:
Yep! I swear I tried that last week, but at any rate that makes sense and it does seem to be defined when doing a `var_dump` from `index.php`.